package/gluon-status-page-mesh-batman-adv/Makefile

```diff
@@ -0,0 +1,23 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-status-page-mesh-batman-adv
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/gluon-status-page-mesh-batman-adv
+  TITLE:=Batman-data provider for gluon-status-page
+  SECTION:=gluon
+  CATEGORY:=Gluon
```

Comment:
The above two lines can be dropped since 994c94918a73a673f25a65fd6795fd89e19decec

Reply:
done